bodhi/server/models/models.py

```diff
@@ -1022,15 +1022,19 @@ def get_cvestring(self):
     def get_bug_karma(self, bug):
         good, bad, seen = 0, 0, set()
         for comment in reversed(self.comments):
-            if comment.user.name in seen:
-                continue
-            seen.add(comment.user.name)
-            for feedback in comment.bug_feedback:
-                if feedback.bug == bug:
-                    if feedback.karma > 0:
-                        good += 1
-                    elif feedback.karma < 0:
-                        bad += 1
+            if comment.user:
```

Comment:
I do believe that comments will always have users:

https://github.com/fedora-infra/bodhi/blob/2.3.3/bodhi/server/models/models.py#L1966

I think we should drop this if statement.